cloud-optimized-geotiffs/accessing-cogs-in-r-terra.ipynb

```diff
@@ -0,0 +1,526 @@
+{
```

Comment:
<p>We should link to the terra documentation. <a href="https://rspatial.github.io/terra" target="_blank">https://rspatial.github.io/terra/</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>